automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -185,6 +189,16 @@ def product_upgrade(product):
                     execute(satellite6_upgrade, host=sat_host)
                 else:
                     execute(satellite6_zstream_upgrade, host=sat_host)
+                # Execute following tasks post upgrade
```

Comment:
Also once you create that function, call it just after 'https://github.com/SatelliteQE/automation-tools/blob/master/automation_tools/satellite6/upgrade/__init__.py#L200'.